sdk/python/kfp/cli/common/colored.py

```diff
@@ -0,0 +1,10 @@
+from termcolor import colored
+
+def print_warning(data):
+  print(colored(data, 'yellow'))
```

Comment:
JFYI: `click` also supports color output: https://github.com/pallets/click/blob/master/examples/colors/colors.py

Reply:
thanks, updated to use click